packages/cashier/src/pages/payment-agent/payment-agent-signup/country-selector/country-selector.tsx

```diff
@@ -17,7 +17,6 @@ type TValues = {
 };
 
 const CountrySelector = ({ onSelect, className }: TCountrySelectorProps) => {
-    const [residence_list, setResidenceList] = React.useState<ResidenceList>();
     const WS = useWS('residence_list');
```

Comment:
@hamid-deriv mmm what do you think about: 
```
const { data: residence_list} = useWS('residence_list');
```
and use `residence_list` further in the code?

I think that it more consistent and more understandable in further code (e.g. when we apply find method for WS.data)

Reply:
It's good idea. :+1:

Reply:
Sorted.